modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/HeadlessDisplayTag.java

```diff
@@ -81,6 +81,10 @@ public CreationMenu getCreationMenu() {
 		return _creationMenu;
 	}
 
+	public Map<String, Object> getEmptyState() {
+		return _emptyState;
+	}
+
```

Comment:
If we move all the added methods and attributes from `HeadlessDisplayTag` to `BaseDisplayTag` we'll get this feature for free in both tags. See for example how [we did with pagination](https://github.com/liferay/liferay-portal/commit/353b58ec49bbdfbf8cb548d20728ac152b546e7b). This is much simpler as there's no processing of the attribute.

We already did this from the beginning with the `uniformActionsDisplay` attribute, [here](https://github.com/liferay/liferay-portal/commit/f79018e8da5a4c2c8f2bd6008b4429d0aec45f10)